README.md

```diff
@@ -1,5 +1,40 @@
-# Cmod A7-35T GPIO <!-- Replace this line with the project name -->
-Created for Vivado 2016.4
+Cmod A7-35T GPIO Demo
+==============
 
-[Link to the project wiki](https://reference.digilentinc.com/learn/programmable-logic/tutorials/cmod-a7-gpio-demo/start)
+Description
+--------------
+This project is a Vivado demo using the Cmod A7's LEDs, RGB LED's, pushbuttons, and USB UART bridge, written in VHDL. When programmed onto the board, the pushbuttons are tied to the LEDs. Every time a button is pressed, the corresponding LED will light up. The RGB LED smoothly transitions between colors.
+
+To use the USB-UART bridge feature of this demo, the Cmod A7-35T must be connected to a serial terminal on the computer it is connected to over the MicroUSB cable. For more information on how to set up and use a serial terminal, such as Tera Term or PuTTY, refer to [this tutorial](https://reference.digilentinc.com/learn/programmable-logic/tutorials/tera-term). On reset, the Cmod A7-35T sends the line “CMOD A7 GPIO/UART DEMO!” to the serial terminal. Whenever one of the buttons is pressed, the line “Button press detected!” is sent.
+
+Requirements
+--------------
+* **Cmod A7-35T**:To purchase a Cmod A7-35T, see the [Digilent Store](https://store.digilentinc.com/cmod-a7-breadboardable-artix-7-fpga-module/)
+* **Vivado 2018.2 Installation**:To set up Vivado, see the [Installing Vivado and Digilent Board Files Tutorial](https://reference.digilentinc.com/vivado/installing-vivado/start).
+* **Serial Terminal Emulator Application**: For more information see the [Installing and Using a Terminal Emulator Tutorial](https://reference.digilentinc.com/learn/programmable-logic/tutorials/tera-term).
+* **MicroUSB Cable**
+ 
+Demo Setup
+--------------
+1. Download and extract the most recent release ZIP archive from this repository's [Releases Page](https://github.com/Digilent/Cmod-A7-35T-GPIO/releases).
```

Comment:
The releases page has not been updated since ~May 2017, eg it doesn't have the release corresponding to the changes pushed to this repo about 8 days ago.

It would also be good if you had instructions on using 'git' from source rather than just grabbing the zip world, if that's possible

Reply:
We are in the process of updating most of our projects to Vivado 2018.2. Here is a link to  instructions on using git from sources:  github.com/digilent/digilent-vivado-scripts.